internal/overlord/servstate/handlers.go

```diff
@@ -609,6 +612,11 @@ func (s *serviceData) stop() error {
 	default:
 		return fmt.Errorf("cannot stop service while %s", s.state)
 	}
+
+	// Tell log manager to stop log forwarding for this service
+	// TODO: should this be in exited() instead?
+	s.manager.logMgr.StopService(s.config.Name)
```

Comment:
don't need this, we just need to pass the stop channel in the initial call to logMgr